util/mod_translation_updater.py

```diff
@@ -425,7 +433,7 @@ def generate_template(folder, mod_name):
 		sources = sorted(list(sources), key=str.lower)
 		newSources = []
 		for i in sources:
-			i = "/".join(os.path.split(i)).lstrip("/")
+			i = "/".join(path_split_all(i)).lstrip("/")
```

Comment:
personally I would skip the theoretical portability here and use something like `i.replace("\\", "/")`

Reply:
That sounds good to me. I was wondering whether that approach is enough.